examples/rtl_433_prometheus_relay.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python3
+
+import collections
+import json
+import os
+import sys
+import threading
+import time
+
+import dateutil.parser
+import http.server
+
+
+class rtl_433(object):
+	# These fields are used to separate and tag unique sensors, instead of being exported
+	# as readings/values.
+	_ID_FIELDS = [
```

Comment:
probably should include `type` and `subtype`

Reply:
Ah! Those were missing on my OS sensors, didn't know. Added.